src/meta-srv/src/handler/node_stat.rs

```diff
@@ -58,6 +58,7 @@ pub struct RegionStat {
     pub approximate_bytes: i64,
     /// Approximate number of rows in this region
     pub approximate_rows: i64,
+    pub table_id: u32,
```

Comment:
I realize that the region id already contains the table id... If the id in this struct means region id.